_redirects

```diff
@@ -0,0 +1,132 @@
+
+/two-years-in-and-its-just-getting-good-26c5c133c4b9 https://getdbt.com/blog/two-years-in-and-it-s-just-getting-good/
+
+/one-analysts-guide-for-going-from-good-to-great-6697e67e37d9 https://getdbt.com/blog/one-analysts-guide-for-going-from-good-to-great/
+
+/how-to-safely-convert-strings-to-integers-in-redshift-a4d7aa39c70 https://getdbt.com/blog/how-to-safely-convert-strings-to-integers-in-redshift/
+
+/on-two-years-of-dbt-6fcdff942d4d https://getdbt.com/blog/on-two-years-of-dbt/
+
+/how-do-you-decide-what-to-model-in-dbt-vs-lookml-dca4c79e2304 https://getdbt.com/blog/how-do-you-decide-what-to-model-in-dbt-vs-lookml/
+
+/how-to-integrate-dbt-and-looker-with-user-attributes-117fa48c1568 https://getdbt.com/blog/how-to-integrate-dbt-and-looker-with-user-attributes/
+
+/using-redshifts-late-binding-views-with-dbt-e735d80f9cfc https://getdbt.com/blog/using-redshift-s-late-binding-views-with-dbt/
+
+/how-to-design-your-looker-explores-171e28465add https://getdbt.com/blog/how-to-design-your-looker-explores/
+
+/the-missing-layers-of-the-analytics-stack-af420e6214bd https://getdbt.com/blog/the-missing-layers-of-the-analytics-stack/
+
+/installing-dbt-packages-83e28210f4e6 https://getdbt.com/blog/installing-dbt-packages/
+
+/whats-new-in-dbt-0-9-0-dd36f3572ac6 https://getdbt.com/blog/what-s-new-in-dbt-0-9-0/
+
+/what-exactly-is-dbt-47ba57309068 https://getdbt.com/blog/what-exactly-is-dbt/
+
+/analytics-playbook-log-tables-7df0e6aacb42 https://getdbt.com/blog/analytics-playbook-log-tables/
+
+/why-doesnt-dbt-support-python-4677caea5cab https://getdbt.com/blog/why-doesn-t-dbt-support-python/
+
+/one-year-in-were-still-in-business-41f959938c70 https://getdbt.com/blog/one-year-in-we-re-still-in-business/
+
+/the-most-underutilized-function-in-sql-9279b536ed1a https://getdbt.com/blog/the-most-underutilized-function-in-sql/
+
+/what-are-the-steps-tools-in-setting-up-a-modern-saas-based-bi-infrastructure-281e0860f9a9 https://getdbt.com/blog/what-are-the-steps-tools-in-setting-up-a-modern-saas-based-bi-infrastructure/
+
+/is-looker-the-right-business-intelligence-tool-for-my-company-afc1f750a0f9 https://getdbt.com/blog/is-looker-the-right-business-intelligence-tool-for-my-company/
+
+/its-time-for-open-source-analytics-194902ae5c5 https://getdbt.com/blog/it-s-time-for-open-source-analytics/
+
+/analyzing-your-mailchimp-data-part-1-measuring-core-email-performance-8c45640155f9 https://getdbt.com/blog/
+
+/analyzing-your-stripe-data-part-1-measuring-subscription-mrr-603d3597e289 https://getdbt.com/blog/modeling-subscription-revenue/
+
+/welcome-youre-now-a-software-developer/ https://getdbt.com/blog/welcome-youre-now-a-software-developer/
+
+# blog.fishtownanalytics.com -> getdbt.com/blog/ redirects
+/author/lauren/ https://www.getdbt.com/author/lauren-craigie/ 301!
```

Comment:
I don't think the "301!" is necessary, since those files don't exist on this site, but no harm in keeping it.